example/README.md

```diff
@@ -155,6 +155,21 @@ Run `terraform apply`.
 
 This will deploy your data-migration resources.
 
+## Choosing a distribution API
```

Comment:
Who are these instructions for specifically? Are these intended to be user facing?

Reply:
If so, I think we should be writing a general upgrade note document (e.g. https://nasa.github.io/cumulus/docs/upgrade-notes/migrate_tea_standalone) about using new Cumulus distribution and how to remove TEA. I don't think we should generally be putting any user facing document in our `example` folder versus the public docs for discoverability.

Reply:
This was only meant for developers deploying from example for CI/testing/etc. There will be user-facing docs that will call out instructions for adapting the template deploy. We'll also note the downtime a switch will cause.

I wanted to get the approach here hammered out first to know what to say there but I'll update that PR as well once we're set.